roles/satellite-receptor/defaults/main.yml

```diff
@@ -1,15 +1,15 @@
 ---
+install_receptor_from_rpm: yes
 satellite_url: "https://satellite.example.com/"
 satellite_user: "fifi_user"
 satellite_password: "0p3ns3s4m3!"
 satellite_ca_file: "/etc/pki/tls/certs/ca-bundle.crt"
 satellite_validate_certs: yes
 receptor_config_dir: "/etc/receptor"
-# TODO: Find a better place for this, something like /var/lib/receptor
-receptor_data_dir: /tmp
+receptor_data_dir: /var/data/receptor
 receptor_packages:
   - receptor
   - python3-receptor-satellite
 c_rh_c_host: "cert.cloud.redhat.com"
-# TODO: Obtain this dynamically?
-source_type_id: "8"
+skip_satellite_org_id_list:
+  - 1
```

Comment:
Why are we skipping org with id 1 by default? (if I'm reading this right)

Reply:
oh, sorry that's my fault of not explaining well, nuc1 setup has forklift manifest there which is problematic, we shoud ignore such error as it's our seutp oddity, we shouldn't be generally skipping org with id 1

Jag if that prevents you from testing, we can probably delete the broken manifest from that org

Reply:
unless this was temporary workaround you planned to remove before merge...

Reply:
Thank you for the reminder :)